modules/odc-deploying-java-applications.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-deploying-java-applications.adoc
+
+[id="odc-deploying-java-applications_{context}"]
+= Upload JAR files for easy deployment of Java applications
+
+You can use the JAR files in the *Topology* view of the *Developer* perspective to deploy your Java applications. You can upload a JAR file using the following options: 
+
+* Navigate to *+Add* view of the *Developer* perspective, click the *Upload JAR file* tile through the *From Local Machine* option. Browse and select your JAR file or drag and drop a JAR file to deploy your application.
+
+* Navigate to the *Topology* view and use the *Upload JAR file* option or drag and drop a JAR file to deploy your application.
+
+* Navigate to the Context Menu in the *Topology* view and use the *Upload JAR file* option to upload your JAR file to deploy your application.
+
+Use the following instructions to upload a JAR file in the *Topology* view to deploy a Java application.
+
+.Procedure
+
+. In the *Topology* view, right-click anywhere in the *Topology* view to see the *Add to Project* menu.
+
+. Hover over the *Add to Project* menu to see the menu options, and then select the *Upload JAR file* option to see the *Upload JAR file* form . Alternatively, you can drag and drop the JAR file in the *Topology* view.
+
+. In the *JAR file* field, browse for the required JAR file on your local machine and upload it. Alternatively, you can drag and drop the JAR file on the field. A toast alert is displayed at the top right if an incompatible file type is dragged and dropped on the *Topology* view. On the other hand, a field error is displayed if an incompatible file type is dropped on the field in the upload form.
```

Comment:
To err on the side of caution, I tend to avoid colloquialisms in official documentation. So instead of on the other hand do you think it'd read just as well to simply say, Or, a field error is displayed if an incompatible file type... ?

Reply:
That's sound advice. Thanks a lot.